src/librbd/CMakeLists.txt

```diff
@@ -30,12 +30,15 @@ set(librbd_internal_srcs
   exclusive_lock/ReleaseRequest.cc
   exclusive_lock/StandardPolicy.cc
   image/CloseRequest.cc
+  image/CreateRequest.cc
   image/OpenRequest.cc
   image/RefreshParentRequest.cc
   image/RefreshRequest.cc
   image/SetSnapRequest.cc
   image_watcher/Notifier.cc
   image_watcher/NotifyLockOwner.cc
+  journal/RemoveJournalRequest.cc
```

Comment:
Minor: since these are in the journal namespace, they could be shortened to just "CreateRequest" and "RemoveRequest" just like all the items in the image, exclusive_lock, etc namespaces.